website/pages/en/resources.js

```diff
@@ -89,36 +74,77 @@ class Resources extends React.Component {
+                  <ResourceCardItem
+                    videolength="Repo"
+                    videotitle="Hermes for Windows"
+                    cardlink="https://github.com/microsoft/hermes-windows"
+                    videotype="Github Repo"
+                    imgurl="./img/homepage/hermes_logo_small.png"
+                    videodisc="Microsoft’s fork of facebook/Hermes that brings Windows support to the lightweight JS engine for React Native."/>
+                    <ResourceCardItem
+                      videolength="Repo"
+                      videotitle="React Native Test App"
+                      videotype="Github Repo Sample"
+                      cardlink="https://github.com/microsoft/react-native-test-app"
+                      imgurl="https://raw.githubusercontent.com/microsoft/react-native-windows/master/.github/hero2.png"
+                      videodisc="React Native Test App provides test apps for all platforms as a package."/>
+             </div>
+             <div className="column">
+              <ResourceCardItem
+                videolength="Repo"
+                videotitle="React Native for macOS"
+                cardlink="https://github.com/Microsoft/react-native-macos"
+                videotype="Github Repo"
+                imgurl="./img/homepage/native_and_js_mac_cropped.png"
+                videodisc="Official React Native for macOS framework for building native macOS applications with React."/>
+                <ResourceCardItem
+                  videolength="Repo"
+                  videotitle="Dual Screen"
+                  videotype="Github Repo"
+                  imgurl="./img/homepage/duo.jpg"
+                  cardlink="https://github.com/microsoft/react-native-dualscreen"
+                  videodisc="Microsoft's offerings to streamline dual-screen cross-platform development using React Native. The modules in the repo are targeting Windows and Android."/>
+                  <ResourceCardItem
+                    videolength="Repo"
+                    videotitle="VS Code RN extension"
+                    videotype="Github Repo"
+                    imgurl="./img/homepage/vs_code_logo.png"
+                    cardlink="https://github.com/Microsoft/vscode-react-native"
+                    videodisc="React Native extension for VS Code enables you to debug your code and quickly and run react-native commands from the command palette."/>
+                    <ResourceCardItem
+                      videolength="Repo"
+                      videotitle="App Center SDK for React Native"
+                      videotype="Github Repo"
+                      imgurl="./img/homepage/appcenter_logo.png"
+                      cardlink="https://github.com/microsoft/appcenter-sdk-react-native"
+                      videodisc="React Native extension for VS Code enables you to debug your code and quickly and run react-native commands from the command palette."/>
```

Comment:
The videodesc field of this one is a copy of VSCode item from previous line. Should update with Appcenter specific description.

Reply:
thanks! fixed.